kalite/topic_tools/content_models.py

```diff
@@ -0,0 +1,389 @@
+import os
```

Comment:
This is, in some ways, the most important, and also the least important part of the refactor.

All of the functions in here are designed to make the database implementation opaque to the rest of the app.

All functions should return a dict, or list of dicts, so that the rest of the app does not have to be concerned with the implementation details of the database backend.

If I have coded it correctly, then every function in here should be repurposable to use a completely different database manager (other than peewee) and the rest of the app could carry on without changes.

What this file needs is complete API documentation, to detail each of the methods, and what their output characteristics should be.